applications/metis_application/custom_processes/set_mpi_communicator_process.h

```diff
@@ -56,7 +56,7 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // Project includes
 #include "includes/define.h"
 #include "includes/model_part.h"
-#include "includes/mpi_communicator.h"
+#include "mpi/includes/mpi_communicator.h"
```

Comment:
@jcotela i think we were discussing about calling this data_communicator... am i wrong? (not a big issue, just that i think i remember you were also in favor of that naming)

Reply:
mmm, seeing that there is actually a data_communicator i am a little lost.

what is the difference between mpi_communicator and data_communictor?

Reply:
`MPICommunicator` and `Communicator` are the the existing `ModelPart`-based objects (with local and ghost meshes, etc.). `DataCommunicator` and `MPIDataCommunicator` are the new wrappers for `MPI_Comm` objects.

I did move the existing `MPICommunicator` to the new MPI core library, but it is the only change I made to that class so far.